angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.scss

```diff
@@ -77,3 +77,9 @@ nimble-table {
 .query-param-word {
     margin-top: $ni-nimble-standard-padding;
 }
+
+.rich-text-viewer-container {
```

Comment:
Is this container always going to be necessary or will it go away when we adopt new styling that more clearly separates the viewer from its background? If it will go away could you make sure the work to delete it is tracked in a work item?

Reply:
I added this container to make the component separated from the background as you mentioned and it is only for the example client app representation. As per the [latest visuals for the nimble viewer component](https://www.figma.com/file/7tTImJ8UueTcmgqcCWmD5q/Rich-Text-Editor-Visuals?node-id=7565%3A31340&mode=dev), the appearance will be transparent and does not have any borders. It is completely based on the client's needs whether to set the border or not.

Reply:
Actually, looking at your Figma link more closely I'm not seeing what you mean about the component not having borders. At least visually it looks like they do. Has something changed or am I missing something?

![image](https://github.com/ni/nimble/assets/10500124/44c91753-4b20-4975-a921-2122a6224719)

Reply:
I meant that the placeholder that is marked inside the red border in the image below, is the specific component I am referring to as `nimble-rich-text-viewer`. Other details such as the username and timestamp will be included as part of the `sl-comment` component ([sl-comment-module HLD](https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/docs/design-documents/Platform/Comments/Sl-Comments-Module-Design.md&version=GBusers/susee/comments-hld/sl-comments-module&line=140&lineEnd=141&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) - currently in PR state.), and the border will encompass both the header (username + timestamp) and the `nimble-rich-text-viewer`.
![image](https://github.com/ni/nimble/assets/123377523/6140fc01-21b4-4462-af19-1dce69ce998b)

Our idea was to make the nimble viewer component borderless by default for now, allowing the client to choose whether to include borders. Later when we incorporate a setting like `appearance`, we will update it accordingly.

Reply:
Ah, right, I'd forgotten about where the boundary was. Given that's our current approved direction then I think what you're doing here makes sense. As we get more clients we should keep an eye on whether the username and timestamp should move into nimble, but that's not a decision we need to revisit now.

Reply:
I don't think we would add specific name and date slots to a generic rich text viewer / editor, but I could imagine requests in the future to have the viewer have an appearance configuration so it looks and behaves more like a form control (like a text area) instead of a block of text (like a p tag). But yea something we can keep an eye on and address in the future.

Reply:
Sure, I will resolve this comment for now and will address this in the future.